src/schema/index.js

```diff
@@ -59,7 +60,7 @@ type syncInfo {
 
 type Query {
   allTopics(filter: TopicFilter, skip: Int, first: Int, orderBy:String): [Topic]!
-  allOracles(filter: OracleFilter, skip: Int, first: Int, orderBy:String): [Oracle]!
+  allOracles(filter: OracleFilter, skip: Int, first: Int, orderBy: [Order]): [Oracle]!
```

Comment:
If this is approved I am going to apply it to all orderBy

Reply:
Makes sense to have reusable generic types for this 👍